modules/log6x-6-2-0-rn.adoc

```diff
@@ -6,52 +6,62 @@
 
 [id="logging-release-notes-6-2-0-technology-preview-features_{context}"]
 == Technology Preview
 
 :FeatureName: The OpenTelemetry Protocol (OTLP) output log forwarder
 include::snippets/technology-preview.adoc[]
 
-////
-* With this update, a `dataModel` field has been added to the `lokiStack` output specification. Set the `dataModel` to `Otel` to configure log forwarding using the OpenTelemetry data format. The default is set to `Viaq`. For information about data mapping see link:https://opentelemetry.io/docs/specs/otlp/[OTLP Specification].
+* With this update, OpenTelemetry support offered by OpenShift {logging-uc} continues to improve, specifically in the area of enabling migrations from the ViaQ data model to OpenTelemetry when forwarding to LokiStack. (link:https://issues.redhat.com/browse/LOG-6146[LOG-6146])
 
-////
+* With this update, the field `structuredMetadata` has been removed from {loki-op} in the `otlp` configuration because structured metadata is now the default type. Additionally, the update introduces a `drop` field that administrators can use to drop OpenTelemetry attributes when receiving data through OpenTelemetry protocol (OTLP). (link:https://issues.redhat.com/browse/LOG-6507[LOG-6507])
 
 [id="logging-release-notes-6-2-0-bug-fixes_{context}"]
 == Bug Fixes
-////
-TOFIX:
-None.
-////
+
+* Before this update, the timestamp shown in the console logs did not match the `@timestamp` field in the message. With this update the timestamp is correctly shown in the console. (link:https://issues.redhat.com/browse/LOG-6222[LOG-6222])
+
+* The introduction of ClusterLogForwarder 6.x modified the ClusterLogForwarder API to allow for a consistent templating mechanism. However, this was not applied to the syslog output spec API for the `facility` and `severity` fields. This update adds the required validation to the ClusterLogForwarder API for the `facility` and `severity` fields. (https://issues.redhat.com/browse/LOG-6661[LOG-6661])
+
+* Before this update, an error in the {loki-op} generating the Loki configuration caused the amount of workers to delete to be zero when `1x.pico` was set as the LokiStack size. With this update, the number of workers to delete is set to 10. (https://issues.redhat.com/browse/LOG-6781[LOG-6781])
+
+[id="logging-release-notes-6-2-0-known-issues_{context}"]
+== Known Issues
+
+* Currently, the logs that were stored using the new OpenTelemetry data model for the LokiStack output, display the following error in the logging Console:
++
+----
+__error__ JSONParserErr 
+__error_details__ Value looks like object, but can't find closing '}' symbol
+----
++
+The error is displayed because the previous data model encoded all information as JSON and the Console still uses the query to parse the information so that both old and new entries are decoded properly.
```

Comment:
```suggestion

```

I moved this part to line no 49 to maintain minimalism. hence suggesting to remove this from here.

Reply:
done